workbench/settings.py

```diff
@@ -97,13 +116,6 @@
 # Make this unique, and don't share it with anybody.
 SECRET_KEY = '5ftdd9(@p)tg&amp;bqv$(^d!63psz9+g+_i5om_e%!32%po2_+%l7'
 
-# List of callables that know how to import templates from various sources.
-TEMPLATE_LOADERS = (
```

Comment:
Maybe you need to add these loaders to `OPTIONS` -> 'loaders' like edx-platform does?:
https://github.com/edx/edx-platform/blob/master/cms/envs/common.py#L314-L317